en/dev_setup/dev_env_windows.md

```diff
@@ -19,23 +19,23 @@ Once you have finished setting up the command-line toolchain:
 - Install [VSCode](../dev_setup/vscode.md) (if you prefer using an IDE to the command line).
 - Install the [QGroundControl Daily Build](https://docs.qgroundcontrol.com/en/releases/daily_builds.html)
   :::tip
-  The *daily build* includes development tools that hidden in release builds. 
+  The *daily build* includes development tools that hidden in release builds.
   It may also provide access to new PX4 features that are not yet supported in release builds.
   :::
 - Continue to the [build instructions](../dev_setup/building_px4.md).
 
 
 ## Other Windows Toolchains
 
-There are a number of other legacy/alternative solutions that may be of interest to some developers. 
+There are a number of other legacy/alternative solutions that may be of interest to some developers.
 A comparison of the options is provided below.
 
 :::note
 The [Cygwin Toolchain](../dev_setup/dev_env_windows_cygwin.md) is the only one that is supported by the PX4 dev team.
-It is regularly tested as part of our continuous integration system and is known to be better performing than the other alternatives. 
+It is regularly tested as part of our continuous integration system and is known to be better performing than the other alternatives.
 :::
 
-| | [Cygwin Toolchain](../dev_setup/dev_env_windows_cygwin.md) **(Supported)**  | [Virtual Machine Toolchain](../dev_setup/dev_env_windows_vm.md) | [Bash on Windows Toolchain](../dev_setup/dev_env_windows_bash_on_win.md) |
+| | [Cygwin Toolchain](../dev_setup/dev_env_windows_cygwin.md) **(Supported)**  | [Virtual Machine Toolchain](../dev_setup/dev_env_windows_vm.md) | [WSL Toolchain](../dev_setup/dev_env_windows_wsl.md) |
```

Comment:
There is a table below here. Do we need to update it. I note that the instructions only cover Pixhawk, Gazebo, Jmavsim, so we should perhaps at least highlight JMAVsim too?

Note that I expect some of the other cases will work too.

Reply:
I have to check if the table is still up to date 👀

Reply:
OK. Update at will.

Reply:
Actually, don't bother. This page is not actually in the sidebar, or built. I magically redirect it (https://docs.px4.io/master/en/dev_setup/dev_env_windows.html) to the cygwin page 

We should delete it, but I kept it in case we reverted the decision to have multiple supported versions.